satellite/audit/reservoir.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Paths []storj.Path
+	Size  int
+	index int
```

Comment:
These can be bytes. And, hardcode the max size as an array.

Reply:
The `Size` can be `byte`, but the `index` will take significantly larger values - up to the total number of pieces per node. I can even argue it should be `int64`, if we want storage nodes to store PB of data.

Reply:
if we hardcode the max size of the array, then the reservoir size won't be configurable unfortunately!

Reply:
It can be configurable up to the specified size.

Reply:
^I'm not totally sure how the config should look/what to implement following this comment.